psi4/driver/procrouting/proc.py

```diff
@@ -1884,6 +1884,11 @@ def scf_helper(name, post_scf=True, **kwargs):
             basisset=scf_wfn.basisset()
         )
 
+
+    # if we pass a jk, set it on the ref_wfn
```

Comment:
1. The comment is not helpful.
2. This seems like a perfect case to use an [assignment expression](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions), since python >= 3.8.

Reply:
Yes, that's what I meant.